### DISCUSSION TITLE
차단한 사람 또 차단하면 에러일까요 성공일까요?

### DISCUSSION THREAD

**Discussion by anso33** [GhostPangPang/GhostPong#89] [General]
문제 상황
차단한 유저를 또 차단하면 에러 응답을 줘야하나요 아니면 차단했다는 성공 응답을 줘야하나요?

발생할 수 있는 시나리오
1. 소현이는 영희를 차단했다 -> 근데 잊어먹고 nickname을 쳐서 차단하는 버튼으로 영희를 또 차단했다. 
2. 소현이는 유저차단 url에 user_id 가 들어가는 것을 보고 아무숫자, 3을 user_id에 넣어서 reqest를 날렸다. -> 그런데 아뿔싸 3은 이미 차단한 영희의 user_id 였다. 

@anso33  @Kimhan-nah  @nyeoni  @jis-kim  @aLVINlEE9

**Comment by Kimhan-nah** [comment]
'이미 차단된 유저입니다'를 알려주면 좋을 것 같아욤
이런 처리들 친구 신청, 신청 수락, 신청 거절, 친구 끊기, 차단, 차단 해제 모든 상황에서 동일하게 확인을 해줄까요??

**Comment by jis-kim** [comment]
전 두 상황 모두 에러 응답이라고 생각합니다! 이미 레코드가 있는 상황이라 `409 Conflict` 정도가 적당할 것 같습니다. 참고로 이미 친구신청 했는데 친구 신청 또 했을 경우는 409 입니다. ㅎㅎ

**Comment by anso33** [comment]
흠냐ㅑ  두분 다 에러쪽이시군요?!!? 좋습니다! 에러로 가시죠
